table/tables/tables.go

```diff
@@ -54,6 +59,9 @@ type Table struct {
 	indexPrefix     kv.Key
 	alloc           autoid.Allocator
 	meta            *model.TableInfo
+
+	// partitionExpr caches partition expression.
```

Comment:
partitionExpr caches the partition definition expressions.

Reply:
ditto

Reply:
so maybe "caches the expressions to locate a partition"?